packages/next/src/builders/build/lib/create-next-config-file.ts

```diff
@@ -0,0 +1,21 @@
+import { copyFile } from 'fs';
+import { promisify } from 'util';
+import { join } from 'path';
+import { NextBuildBuilderOptions } from '../../../utils/types';
+
+const asyncCopyFile = promisify(copyFile);
+
+export async function createNextConfigFile(options: NextBuildBuilderOptions) {
+  const nextConfigPath = join(
+    options.root,
+    options.nextConfig || 'next.config.js'
```

Comment:
This is relative to the project root right?

Reply:
No, you want the context to be passed in so you can  do this: `join(context.workspaceRoot, options.root);`